config/common_config.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package config
+
+import (
+	"crypto/tls"
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanchego/staking"
+	"github.com/spf13/viper"
+)
+
+const TLSCertPathKey = "tls-cert-path"
+const TLSKeyPathKey = "tls-key-path"
+
+func GetTLSCertFromFile(v *viper.Viper) (tls.Certificate, error) {
```

Comment:
I feel like this should return `(*tls.Certificate, error)`. `LoadTLSCertFromFiles` returns a pointer and our one call to this function converts what we're currently returning into a pointer.